app/controllers/application_controller.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  helper_method :current_user
+
+  def current_user
+    false
```

Comment:
when the time comes, we can fill this in with the appropriate code. Just hardcoded `false` to see what the login/register buttons look like, and then changed it to `true` to see logout.

Reply:
looks good nice work